app/src/main/java/ch/epfl/qedit/backend/Firebase.java

```diff
@@ -0,0 +1,15 @@
+package ch.epfl.qedit.backend;
+
+import ch.epfl.qedit.frontendBackendInterface.Request;
+import ch.epfl.qedit.frontendBackendInterface.Response;
+
+public class Firebase extends Backend {
+    public Firebase() {
```

Comment:
Wasn't it supposed to be my job to do that ?

Reply:
Yes but as you can see this class is pretty much empty. It's just a stub, a placeholder, for me that made it a bit easier for me to actually do something. When merging your code will be put inside this class.

Reply:
> Because we want to make the application independent of the backend running behind it. We abstract it away.

But the goal is to hide Firebase and not showing that we use Firebase. What about abstracting it now ?

> When merging, your code will be put inside this class.

You're kidding ? The goal is to hide Firebase behind an adapter (`Server` in the already merged code). Not show it to our application that we used Firebase.

Delete that awful class.

Reply:
No need to get toxic. "Backend" plays to role of an adapter or at least it will once we have your code.

Reply:
Or delete that class and then make the `Server` class that I have already created inherit from Backend. No need for anyone else in the app to be aware that Firebase exists.

Reply:
Yes that is a good option. Firebase is or was the equivalent to Server.